config/longrun_configs/slabplanet_aqua_target_evolve_ocn.yml

```diff
@@ -9,7 +9,7 @@ energy_check: false
 evolving_ocean: true
 land_albedo_type: "function"
 mode_name: "slabplanet_aqua"
-mono_surface: false
+
```

Comment:
Do you want to add a blank line to all these files?

Reply:
no 😅